app/frontend/src/wells/components/SearchResultExports.vue

```diff
@@ -71,23 +85,31 @@ export default {
 
       return querystring.stringify(queryParams)
     },
-    excelExportUrl () {
-      return this.getExportUrl('xlsx')
-    },
-    csvExportUrl () {
-      return this.getExportUrl('csv')
-    }
   },
   methods: {
+    /**
```

Comment:
Should we resolve the duplication in the the APIService?

Reply:
@fergmac 
The misleading `${process.env.VUE_APP_AXIOS_BASE_URL}` === '/api' the whole time. 
Removed and resolved.